client/src/components/Footer.js

```diff
@@ -5,6 +5,9 @@ import { Link } from 'react-router';
 const Footer = () => (
   <Grid fluid className="footer-container">
     <Col xs={12}>
+      <p>Questions? Reach out to us!</p>
```

Comment:
This font could be a bit smaller? It's larger than some of the body font above.

Reply:
Totally agree - changed to 14px.